python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -39,10 +41,13 @@ class Translator:
     ----------
     visitor
         Polars NodeTraverser object
+    config
+        GPU engine configuration.
     """
 
-    def __init__(self, visitor: NodeTraverser):
+    def __init__(self, visitor: NodeTraverser, config: GPUEngine):
```

Comment:
Wasn't sure if we want `config` to be a required argument. I suppose we could also fall back to `pl.GPUEngine()` if nothing is passed in (would simplify any testing and debugging cases where a special configuration isn't needed).

Reply:
My intuition is that it should be explicit as we have here already. I feel like instantiating configs by default too far into our internals might make it harder to figure out where the config being observed is came from in the first place.